lib/executor/common_test.go

```diff
@@ -79,6 +79,6 @@ func initializeVUs(
 	for i := uint64(0); i < number; i++ {
 		vu, err := es.InitializeNewVU(ctx, logEntry)
 		require.NoError(t, err)
-		es.AddInitializedVU(vu)
+		es.ReturnVU(vu, false)
```

Comment:
This is because AddInitializedVU adds 1 to the number of initializedVus .. this also seemed like the only place it is used, so maybe we can remove it :)

Reply:
It turns out that this broke the `externally-controlled` tests, so I reverted it. Also, it's used here: https://github.com/loadimpact/k6/blob/447f892cc2c4073290e25b220307851d14aaae95/core/local/local.go#L203